wallet/src/wallet/error.rs

```diff
@@ -127,12 +127,12 @@ impl fmt::Display for CreateTxError {
                 requested,
                 required,
             } => {
-                write!(f, "TxBuilder requested timelock of `{:?}`, but at least `{:?}` is required to spend from this script", required, requested)
+                write!(f, "TxBuilder requested timelock of `{}`, but at least `{}` is required to spend from this script", required, requested)
```

Comment:
The required/requested values are swapped, aren't they? (it looks like they're already previously)